extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -398,7 +398,8 @@ public function getCommunityHeaderBackgroundUrl(): string {
 		$thumbnailUrl = '';
 		$originalUrl = $this->getSettings()['community-header-background-image'];
 
-		if ( VignetteRequest::isVignetteUrl( $originalUrl ) ) {
+		// @todo fix the issue with vignette urls without /latest and remove second condition
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)